wrapper/src/test/java/integration/refactored/host/TestRunner.java

```diff
@@ -16,6 +16,9 @@
 
 package integration.refactored.host;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import integration.refactored.TestEnvironmentFeatures;
```

Comment:
nit: are these needed if there are no changes to the file? Or are they unused imports?